app/domain/importers/content_details.rb

```
@@ -13,8 +13,29 @@ def initialize(content_id, base_path)
 
   def run
     response = items_service.fetch_raw_json(base_path)
-    number_of_pdfs = Performance::Metrics::NumberOfPdfs.parse(response.to_h['details'])
+    metadata = format_metadata(response.to_h)
+    attributes =  metadata.merge(raw_json: response)
     item = Dimensions::Item.find_by(content_id: content_id, latest: true)
-    item.update_attributes(raw_json: response, number_of_pdfs: number_of_pdfs)
+    item.update_attributes(attributes)
```

I would use `update_attributes!` so errors don't silently fail.